microprot/scripts/processing.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+__author__ = "Tomasz Kosciolek"
```

Comment:
I followed / respected the original coding style established by @tkosciol . But I think a better way would be to say  "The MicroProt Development Team", just like what we did in [WGS-HGT](https://github.com/biocore/WGS-HGT/blob/master/benchmark/parse_output.py).